docker/build_bionic/Dockerfile

```
@@ -0,0 +1,27 @@
+# A new image must to be pushed, to make changes to this file valid:
+# docker build -t registry.gitlab.com/fritzing/fritzing-app_playground/build:bionic docker/build_bionic
+# docker push registry.gitlab.com/fritzing/fritzing-app_playground/build:bionic
+
+FROM ubuntu:bionic-20190307
+
+MAINTAINER Kjell kjell@gmx.de
```

This field is deprecated. https://docs.docker.com/engine/reference/builder/#maintainer-deprecated
Please use `LABEL maintainer="foo@bar.com"` instead.

```suggestion
LABEL maintainer="Kjell kjell@gmx.de"
```